packages/core/src/lib/database/migrations/1748354102433-AlterRegistryTables.ts

```
@@ -0,0 +1,605 @@
+import { MigrationInterface, QueryRunner } from 'typeorm';
+
+export class AlterRegistryTables1748354102433 implements MigrationInterface {
+	name = 'AlterRegistryTables1748354102433';
+
+	/**
+	 * Up Migration
+	 *
+	 * @param queryRunner
+	 */
+	public async up(queryRunner: QueryRunner): Promise<void> {
+		console.log(chalk.yellow(this.name + ' start running!'));
```

Biome (1.9.4)

[error] Template literals are preferred over string concatenation.

Unsafe fix: Use a template literal.


(lint/style/useTemplate)